docs/blog/_posts/2021-02-17-scala3-rc1.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: blog-page
+title: Scala 3.0.0-RC1 – first release candidate is here
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-02-17
+---
+
+Greetings from the Scala 3 team! We are delighted to announce the first release candidate of the stable version of Scala 3 – Scala 3.0.0-RC1.
+
+This release brings some last-minute polishings, clean-ups and changes before the big release. There were a few language changes to improve the user experience, as well as the polishings of the metaprogramming framework. We have also worked on the issues that had to be fixed before the stable release.
+
+Overall, more than [400 PRs](https://github.com/lampepfl/dotty/pulls?q=is%3Apr+is%3Aclosed+closed%3A%3E2020-12-02+sort%3Acomments-desc) were merged after the M3 release and until today! Read more below!
+
+<!--more-->
+# Allow leading context parameters in extension methods
+Type parameters on extensions can now be combined with type parameters on the methods themselves. E.g.:
+
+```scala
+List(1, 2, 3).second[Int]
+extension [T](xs: List[T])
```

Comment:
```suggestion
extension [A](xs: List[A])
```

Reply:
Or some other combination. Maybe it should be `T=>U` for the function instead of `A=>B`